outputs/policyreport.go

```diff
@@ -166,17 +186,28 @@ func updatePolicyReports(c *Client, namespace string, event *wgpolicy.PolicyRepo
 			ObjectMeta: metav1.ObjectMeta{
 				Name: policyReportBaseName + uuid.NewString()[:8],
```

Comment:
Maybe one thing in addition, you could also add a label `managed-by:  "falcosidekick"` or something similar. Kyverno does that and I also in my adapters. So you cal also select only reports created by falcosidekick.

Reply:
```
apiVersion: wgpolicyk8s.io/v1alpha2                                                                                                                                                                                
kind: PolicyReport                                                                                                                                                                                                 
metadata:                                                                                                                                                                                                          
  creationTimestamp: "2022-07-22T08:33:50Z"                                                                                                                                                                        
  generation: 1                                                                                                                                                                                                    
  labels:                                                                                                                                                                                                          
    app.kubernetes.io/managed-by: falcosidekick                                                                                                                                                                    
  name: falco-policy-report-12d01ab7                                                                                                                                                                               
  namespace: default                                                                                                                                                                                               
  ownerReferences:                                                                                                                                                                                                 
  - apiVersion: v1                                                                                                                                                                                                 
    controller: false                                                                                                                                                                                              
    kind: Namespace                                                                                                                                                                                                
    name: falco                                                                                                                                                                                                    
    uid: 8247029e-ba31-4de6-8748-c2bea5437498  
```